README.md

```diff
@@ -162,19 +185,33 @@ class ViewController: UIViewController {
         super.viewDidLoad()
     }
 
-    private func validateSNAURL() async {
-        let snaUrlFromBackend = await asyncMethodToGetSNAUrl()
-
-        twilioVerify.processURL(snaUrlFromBackend) { result in
-            switch result {
-                case .success:
-                // Handle success scenario
-
-                case .failure(let error):
-                // Handle error scenario
+    private func validateSNAURL() {
+        // First check for cellular connectivity
+        twilioVerify.testConnectivity(to: "apple.com", port: 80) { [weak self] isConnected in
```

Comment:
Should/Could this be part of `processURL`? It seems more steps for the customer, so we could test connectivity with the url received from `getBackendSNAUrl` or even calling `https://verify.twilio.com/`, wdyt?

Reply:
As the connection is mandatory for SNA to work, can this be part of the SNA process? We could throw an error if the device does not have a connection, using the SNA url somehow (not sure if it is possible to execute a HEAD to not consume the url). Another option could be removing the address and port and using something in our end (like verify endpoint)